src/lib/registration/widgets/package_search.rb

```diff
@@ -201,16 +247,72 @@ def update_details
 
       # Asks the user to enable the addon
       #
-      # It omits the question if the addon is already registered or selected for registration.
-      #
       # @param addon [Addon] Addon to ask about
       def enable_addon?(addon)
+        description = Yast::HTML.Para(
+          format(
+            _("The selected package is provided by the '%{name}' module/extension, " \
+              "which is not enabled on this system yet."),
+            name: addon.name
+          )
+        )
+
+        unselected_deps = addon.dependencies.reject { |d| d.selected? || d.registered? }
+        if !unselected_deps.empty?
+          description << Yast::HTML.Para(
+            format(
+              _("Additionally, '%{name}' depends on the following modules/extensions:"),
+              name: addon.name
+            )
+          )
+          description << Yast::HTML.List(unselected_deps.map(&:name))
+        end
+        question = n_(
+          "Do you want to enable it?", "Do you want to enable them?", unselected_deps.size + 1
+        )
+        yes_no_popup(description + question)
+      end
+
+      # Asks the user to disable the addon
+      #
+      # @param addon [Addon] Addon to ask about
+      def disable_addon?(addon)
         message = format(
-          _("'%{name}' module/extension is not enabled for this system.\n" \
-            "Do you want to enable it?"),
+          _("The '%{name}' module/extension is not needed anymore.\n" \
```

Comment:
```suggestion
          _("The '%{name}' is not needed anymore.\n" \
```

Again, with possibly some `TRANSLATORS` comment...